app/src/controllers/user/selectors.js

```diff
@@ -8,6 +8,7 @@ export const userIdSelector = (state) => userInfoSelector(state).userId;
 export const settingsSelector = (state) => userSelector(state).settings;
 export const startTimeFormatSelector = (state) => settingsSelector(state).startTimeFormat;
 export const assignedProjectsSelector = (state) => userInfoSelector(state).assigned_projects || {};
+export const userAccountSelector = (state) => userInfoSelector(state).userRole || '';
```

Comment:
userRoleSelector

Reply:
misprint from my side. Better name is userAccountRoleSelector